README.md

```diff
@@ -1,44 +1,50 @@
-# PureScript API
+# Try PureScript
 
 [![Build Status](https://api.travis-ci.org/purescript/trypurescript.svg?branch=master)](http://travis-ci.org/purescript/trypurescript)
 
-Very basic web service which wraps the PureScript compiler.
+[Try PureScript](https://try.purescript.org) is an online PureScript code editor for quickly experimenting with PureScript code snippets and ideas. It consists of a client and a server component, both of which live within this repository.
 
-[Client code](https://github.com/purescript/trypurescript/tree/gh-pages)
+## Features:
 
-## API
+- Writing code using the [Ace Editor](http://ace.c9.io)
+- Automatic compilation
+- PureScript syntax highlighting
+- Run and print output or show resulting JavaScript
+- Multiple view modes: code, output or both
+- Persistent session
+- Load PureScript code from Github Gists
 
-### Compile PureScript code
+### Control Features via the Query String
 
-**POST /compile**
+Most of these features can be controlled not only from the toolbar, but also using the [query parameters](https://en.wikipedia.org/wiki/Query_string):
 
-- Request body: PureScript code
-- Response body: Either `{ js: "..." }` or `{ error: "..." }`
-- Status code: 200 (success)
+- **Load From Gist**: Load PureScript code from Gist id using the `gist` parameter
+    - Example: `gist=37c3c97f47a43f20c548` will load the code from this Gist if the file was named `Main.purs`
 
-The response does not use error codes, to make it easier to use the API from another domain using CORS.
+- **View Mode**: Control the view mode using the `view` parameter
```

Comment:
We should probably include the default for each of these, ie.

```
Control the view mode using the `view` parameter.

- Options are: `code` | `output` | `both`
- Default is: `both`
- Example: `view=output` will only display the output
```

You could judge this for yourself by loading up the default Try PureScript page, but if you're say crafting a query string based off of this section in the readme it would be useful to see which options you can skip.